### ISSUE TITLE
crash when scanning `MRSpectroscopyStorage` image

### ISSUE THREAD

**Issue by rkm** [SMI/IsIdentifiable#455]
These shouldn't be scanned for Pixel Data, as they don't have any!

We should probably amend this:

https://github.com/SMI/IsIdentifiable/blob/main/IsIdentifiable/Runners/DicomFileRunner.cs#L235

Using the information fo-dicom provides on the `SOPClassUID` here:

https://github.com/fo-dicom/fo-dicom/blob/development/FO-DICOM.Core/DicomUID.cs#L31-L43

**Comment by howff** [comment]
Maybe just check for the PixelData tag being present?

If IsIdentifiable can also scan overlays then check for any of the 16 OverlayData tags being present?

**Comment by rkm** [comment]
Fair point 😉 I guess the question is whether we want to detect invalid files based on whether the SOPClass specifies pixel data _should_ be present?

Not sure on the overlay handling - will check this.
